AsyncDisplayKit/Details/ASRangeController.mm

```diff
@@ -130,8 +138,7 @@ - (void)_updateVisibleNodeIndexPaths
   NSMutableOrderedSet<NSIndexPath *> *allIndexPaths = [[NSMutableOrderedSet alloc] initWithSet:visibleIndexPaths];
   
   ASInterfaceState selfInterfaceState = [_dataSource interfaceStateForRangeController:self];
-  ASLayoutRangeMode rangeMode = [ASRangeController rangeModeForInterfaceState:selfInterfaceState
-                                                                 currentRangeMode:_currentRangeMode];
+  ASLayoutRangeMode rangeMode = (_currentRangeMode == ASLayoutRangeModeInvalid) ? ASLayoutRangeModeMinimum : _currentRangeMode;
```

Comment:
would be slightly easier to read as-
rangeMode = _currentRangeMode
if (rangeMode == invalid)
rangeMode = minimum